### ISSUE TITLE
part 1 instructions say to clone saasbook repo

### ISSUE THREAD

**Issue by mverdicchio** [saasbook/hw-sinatra-saas-hangperson#53]
intro instructions say to fork then to clone the fork but the part 1 commands are not consistent with this

**Comment by tansaku** [comment]
yeah, I keep thinking the forking isn't the way to go and that it's preferrable for students to just clone the repo in order to avoid them accidentally sharing their solutions by pushing them back up to github. I think others prefer that students fork them - of course the instructions should be consistent.

With developed github skills the order of forking and cloning is irrelevant, since you can clone from the main repo and then connect to your forked repo later (using the 'git remote') commands.

Nothing about how any of the assignments is assessed requires them to be forked, although some instructors may require their students to push the assignments to a fork in order to view them.  Ideally we'd adjust the grader so that submission was via student's forked repos, but that's a change that would require a fair chunk of work ...
